components/config-mgmt-service/backend/elastic/nodes.go

```diff
@@ -100,18 +100,18 @@ func (es Backend) GetInventoryNodes(ctx context.Context, start time.Time,
 	return nodes, nil
 }
 
-func (es Backend) GetNodesPageByCurser(ctx context.Context, start time.Time,
-	end time.Time, filters map[string][]string, cursorField interface{},
+func (es Backend) GetNodesPageByCursor(ctx context.Context, start time.Time,
+	end time.Time, filters map[string][]string, cursorValue interface{},
```

Comment:
nitpick: This code uses the term `field` to refer to eg a field on a struct, so using `field` to refer to the value assigned to a field on a struct was confusing to me.